Interchain Exchange review (#577)
cosmos/sdk-tutorials: interchain-exchange/tutorial/00-intro.md

# Introduction

The Interchain Exchange is a module to create buy and sell orders in between blockchains.

> In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains.

Reviewer: ```suggestion
In this tutorial, you will learn how to create a Cosmos SDK module that can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.
```

Author: We always use `orderbook` as one-word. I would like to stay consistent with this wording.

Reviewer: Shouldn't we change it everywhere in the future? I mean I don't think orderbook is grammatically correct

Author: We can use `order book` and then consistent. I can make a PR with that change tomorrow

> In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.

Reviewer: `which enables to swap tokens from one blockchain to another` why removing this statement? This was pretty cool

Author: Ok, great. I thought it might create other expectations. But if you approve I love to keep it 😁

Reviewer: Well it's actually about token swap
